use-cases/index.markdown

```diff
@@ -0,0 +1,21 @@
+---
+layout: default
+title: Use Cases
+permalink: /use-cases/
+---
+
+<div class="row justify-content-center">
```

Comment:
Not a big fan of having a markdown file containing only HTML. Could this be moved to a layout file instead? (Similar to [_layouts/quickstarts.html](https://github.com/kroxylicious/kroxylicious.github.io/blob/main/_layouts/quickstart.html), in this case you would only need to alter the front matter above to move the title and permalink to a seperate markdown file.)

Reply:
I'll give that a go